.github/workflows/automerge.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow will merge pull requests automatically
+#
+
+name: automerge
+on:
+  check_suite:
+    types:
+      - completed
+jobs:
+  automerge:
+    if: github.event.check_suite.app.name == 'Azure Pipelines' &&  github.event.check_suite.conclusion == 'success'
```

Comment:
>  [](start = 65, length = 1)

one extra blank

Reply:
Fixed.

Reply:
Sorry, I mean you have one extra blank after `&&`

---
In reply to: [607427677](https://github.com/Azure/sonic-buildimage/pull/7065#discussion_r607427677) [](ancestors = 607427677)